.github/workflows/auto-update-npm-libraries.yml

```diff
@@ -35,15 +35,6 @@ jobs:
           npm run eslint
           git add package.json package-lock.json
 
-      - name: update paella-player test libraries
```

Comment:
There's no equivalent stanza here for paella 7.  Should there be?

Reply:
There could be, but Dependabot will pick that up.
Would reduce the Dependabot pull requests though.